xml/cap_admin_backup-restore.xml

```diff
@@ -521,5 +523,249 @@
+     </para>
+<screen>&prompt.user;tar cfvz <replaceable>blobstore-src.tgz</replaceable> /var/vcap/store/shared</screen>
+<screen>&prompt.user;exit</screen>
+    </step>
+    <step>
+     <para>
+      Copy the archive to a location outside of the pod:
+     </para>
+<screen>&prompt.user;kubectl cp scf/blobstore-0:blobstore-src.tar.gz <replaceable>/tmp/blobstore-src.tgz</replaceable></screen>
+    </step>
+    <step>
+     <para>
+      Export the Cloud Controller Database (CCDB) into a file:
+     </para>
+<screen>&prompt.user;kubectl exec mysql-0 --namespace scf -- bash -c \
+  '/var/vcap/packages/mariadb/bin/mysqldump \
+  --defaults-file=/var/vcap/jobs/mysql/config/mylogin.cnf \
+  ccdb' > <replaceable>/tmp/ccdb-src.sql</replaceable>
+</screen>
+    </step>
+    <step xml:id="get-encryption-key">
+     <para>
+      Next, obtain the CCDB encryption key(s). The method used to capture the key will depend on whether <literal>current_key_label</literal> has been defined on the source cluster. This value is defined in <filename>/var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml</filename> of the <literal>api-group-0</literal> pod and also found in various tables of the MySQL database.
+     </para>
+     <para>
+      Begin by examining the config file for the<literal>current_key_label</literal> setting:
+     </para>
+<screen>&prompt.user;kubectl exec -it --namespace scf api-group-0 -- bash -c "cat /var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml | grep -A 3 database_encryption"</screen>
+     <itemizedlist>
+      <listitem>
+       <para>
+        If the output contains the <literal>current_key_label</literal> setting, save the output for the restoration process. Adjust the <command>-A</command> flag as needed to include all keys.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        If the output does not contain the <literal>current_key_label</literal> setting, run the following command and save the output for the restoration process:
+       </para>
+      <screen>&prompt.user;kubectl exec api-0 --namespace scf -- bash -c 'echo $DB_ENCRYPTION_KEY'</screen>
+      </listitem>
+     </itemizedlist>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.cap.raw-data-restore-procedure">
+   <title>Performing a raw data restore</title>
+   <para>
+    Perform the following steps to restore your backed up data to the target SCF deployment.
+   </para>
+   <important>
+    <title>Ensure access to the correct SCF deployment</title>
+     <para>
+      Working with multiple &kube; clusters simultaneously can be confusing. Ensure you're communicating with the desired cluster <link xlink:href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable">by setting <literal>$KUBECONFIG</literal> correctly</link>.
+     </para>
```

Comment:
you are, not you're

Reply:
Addressed in https://github.com/SUSE/doc-cap/pull/280/commits/fcde67d3da1a19b5743da5d0fb77df36f5e37f01